client/src/components/form/TokenCredentials.jsx

```diff
@@ -145,6 +145,7 @@ export default function TokenCredentials(props) {
         </div>
       ) : credFormat === "json" ? (
         <Field
+          inputProps={{ "data-testid": "credentials-field" }}
```

Comment:
Could you add a comment that mentions that inputProps & data-testid  is being used for the tests for all of the components? This is just to ensure that it's clear what they are being used for.

Reply:
Fixed.